src/compiler/utilities.ts

```diff
@@ -6674,6 +6674,25 @@ namespace ts {
         return node.kind === SyntaxKind.NamedImports || node.kind === SyntaxKind.NamedExports;
     }
 
+    /* @internal */
+    /**
+     * Strip off existed single quotes or double quotes from a given string
+     *
+     * @return non-quoted string
+     */
+    export function stripQuotes(name: string) {

```

Comment:
I feel like we should just be able to call `symbolToString` and get the right thing in certain cases, and just funnel along a `string`.